src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx

```diff
@@ -432,6 +434,16 @@ export const getMapBounds = (jurisdiction: Jurisdiction | null) => {
   return mapBounds;
 };
 
+/** dynamically get the bounds and zoom for the maps viewport.
+ * ref: https://github.com/mapbox/mapbox-gl-js/issues/1970#issuecomment-297465871
+ */
+export const setMapViewPortZoomFactory = (mapBounds: BoundingBox) => (map: Map) => {
+  const mapEl = map.getCanvas().getBoundingClientRect();
+  const mapDim: [number, number] = [mapEl.height, mapEl.width];
+  const newBounds = viewport(mapBounds, mapDim, 0, 20, 512);
+  map.zoomTo(newBounds.zoom, { duration: 1000 });
```

Comment:
Could we turn these numbers into named constants?  Don't think we should move to the constants module (yet) but lets name them for easier reading of the coe